documents/operating.md

```diff
@@ -0,0 +1,216 @@
+Bad Idea Factory Limited Liability Corporation
+===================================================================
+
+Contents:
+
+1.  Official Operating Agreement
+2.  Partnership
+3.  Projects
+4.  Money
+5.  Governing Process
+6.  Policies
+
+---------------------------------------------------------------------
+SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+---------------------------------------------------------------------
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?)
+or (3) everyone dies / there are no more Corporate Overlords.
+
+### Continuance of Company
+If a Corporate Overlord leaves BIFFUD, dies, or uploads their brain to the
+cloud, the ownership of the remaining corporate overlords will then be evenly
+distributed (e.g. if four overlords remain, each will own 25% of BIFFUD)
+
+### Lawsuits between BIFFUD associates
+All BIFFUD associates involved in a lawsuit mustplay a group game of
+Mario Party 3 before any arbitration happens.  BIFFUD will loan any
+necessary hardware needed to make this possible.
```

Comment:
By "necessary hardware" does this include emulators or should it be a requirement that it be on an original Nintendo 64?